proposals/solid-oidc/index.bs

```diff
@@ -270,11 +270,12 @@ claim redundant, so in Solid-OIDC the `aud` claim MUST be a string with the valu
     {
         "webid": "https://janedoe.com/web#id",
         "iss": "https://idp.example.com",
+        "sub": "janedoe",
```

Comment:
Line 251, add an explanation for the sub claim?

```markdown
  `sub` - OPTIONAL. The subject claim MUST  be a unique identifier for the user within the IdP identified by the issuer's claim.
```

Reply:
Or should the `sub`claim be required as well?

Reply:
I don't know why we would need to say anything beyond what JWT and OIDC already state about `sub`. According to OIDC, `sub` is [required in the ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken), so making it optional would conflict with OIDC. We also don't state anything about `iss` or `exp`.

Reply:
This is the Access token described here and `iss` is described line 252 and `exp` line 259, hence my suggestion for the sake of consistency.

The ID Token ammendment to the `sub` claim is below line 296.

Reply:
```suggestion
```

Per the conversation, I take it that the `sub` claim is required in the ID Token, not the Access Token that is described here. I would suggest just remove that line.

Otherwise all looks good to me and I would suggest merging.

Reply:
```suggestion
```

Per the conversation, I take it that the sub claim is required in the ID Token, not the Access Token that is described here. I would suggest just remove that line.

Otherwise all looks good to me and I would suggest merging.